.cloudcannon/content_blocks.cloudcannon.structures.yml

```
@@ -0,0 +1,8 @@
+contentBlocks:
+  values_from_glob:
+    - /src/components/pricing/table/*.cloudcannon.structure-value.yml
+    - /src/components/home/video/*.cloudcannon.structure-value.yml
+    - /src/components/contact/form/*.cloudcannon.structure-value.yml
+    - /src/components/global/**/*.cloudcannon.structure-value.yml
+    - /src/components/about/video/*.cloudcannon.structure-value.yml
+    - /src/components/about/team/*.cloudcannon.structure-value.yml
```

I'm not sure if these are all split out for a reason, but you could probably condense these down to something like:
```suggestion
    - /src/components/**/*.cloudcannon.structure-value.yml
```